pkg/handler/repositories.go

```diff
@@ -222,3 +275,91 @@ func (rh *RepositoryHandler) deleteRepository(c echo.Context) error {
 	}
 	return c.NoContent(http.StatusNoContent)
 }
+
+//
+// Helper methods
+//
+
+func (rh *RepositoryHandler) mapRepositoryResponse2EventMessage(input *api.RepositoryResponse) (*message.IntrospectRequestMessage, error) {
+	if input == nil {
+		return nil, fmt.Errorf("input cannot be nil")
+	}
+	output := &message.IntrospectRequestMessage{
```

Comment:
I know for this PR it is, but is an `IntrospectRequestMessage` the only message we would send to Kafka in the future? I'm not sure how, but I wonder if this method and the `produceMessage` method should somehow handle messages more generically.

Reply:
Maybe something like, you take the `RepositoryResponse` and then pass it to a method from the `event` (or similar) package that figures out what to do with the response. Kinda like the dao layer but in this case it's kafka.

Reply:
I don't forget this; thinking about it.

Reply:
I have put the adapter code at pkg.event.adapter ; I have tried to provide some semantic through the usage of the package name; Every adapter for the event message will be into an interface related with the message (idiomatic, not a direct relationship of `has ...` or `is ....`).